### PR TITLE
DDF-1569 Removed configuration page and added a config info page.

### PR DESCRIPTION
**Pull request by clockard**
@shaundmorris @andrewkfiedler 
This change should only be merged after https://github.com/codice/ddf/pull/248 is merged.

This is a temporary fix until we get the full installation config pages added.

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/257)

### CODE REVIEW

**Comment by andrewkfiedler** [comment]
:+1: built and tested

File: platform/admin/ui/src/main/webapp/js/views/installer/Configuration.view.js [hunk 1]
```diff
@@ -18,169 +18,28 @@ define([
     'marionette',
     'icanhaz',
     'underscore',
+    'jquery',
```
Comment (roelens8) [inline platform/admin/ui/src/main/webapp/js/views/installer/Configuration.view.js]: Looks like this isn't needed.

Comment (clockard) [inline platform/admin/ui/src/main/webapp/js/views/installer/Configuration.view.js]: You are correct. Removed.

**Comment by tbatie** [comment]
LGTM, did a quickbuild and checked it out as well.

**Comment by roelens8** [comment]
LGTM :+1:

File: platform/admin/ui/src/main/webapp/templates/installer/configuration.handlebars [hunk 2]
```diff
@@ -11,7 +11,16 @@
  *
  **/
  --}}
-<h3>Configure general settings</h3>
+<h3>Additional System Settings</h3>
 <form id="config-form" class="inner-scroll scroll-area ps-container" method="GET">
+    <div class="well white">
+        <p class="lead">
+            <br/>
+            After installation is complete there are addition configurations that can be made
```
Comment (shaundmorris) [inline platform/admin/ui/src/main/webapp/templates/installer/configuration.handlebars]: "additional" configurations?

**Pull request merged**